src/core_plugins/kibana/public/management/sections/settings/advanced_row.html

```diff
@@ -118,6 +137,12 @@
           ng-if="conf.markdown"
           ng-bind-html="conf.value | markdown"
         ></span>
+        <img
+          alt="Image"
+          ng-if="conf.image && conf.value"
+          src="{{ conf.value }}"
```

Comment:
I think we need some more validation here,
```
PUT .kibana/doc/config:7.0.0-alpha1
{
  "config": {
    "test": "https://site-with-basic-auth"
  }
}
```
will show a basic auth prompt in non chrome browsers.

Reply:
Good catch!! I'll find a way to make sure that it's a data-url.